### PR TITLE
Undefined variables

### PR DESCRIPTION
**Pull request by jdalsem**
and some other notices issues...

### CODE REVIEW

**Comment by jdalsem** [comment]
Simple... but fixes 7 issues

File: engine/classes/Elgg/Debug/Inspector.php [hunk 1]
```diff
@@ -167,10 +167,10 @@ public function getActions() {
 	 * @return array [views]
 	 */
 	public function getSimpleCache() {
-		global $CONFIG;
-
+		
+		$simplecache = elgg_extract('simplecache', $this->getViewsData(), []);
 		$tree = array();
-		foreach ($CONFIG->views->simplecache as $view => $foo) {
+		foreach ($simplecache as $view => $foo) {
```
Comment (mrclay) [inline engine/classes/Elgg/Debug/Inspector.php]: This also needs to add in all non-PHP view files (see the other PR)

Comment (mrclay) [inline engine/classes/Elgg/Debug/Inspector.php]: Add this in https://github.com/Elgg/Elgg/pull/9668/files#diff-4309aeabc90256cf63cdd7c29cc8deebR176

**Comment by mrclay** [comment]
LGTM besides what I mentioned

**Comment by hypeJunction** [comment]
Could you merge this up before it creates too many conflicts.

**Comment by jdalsem** [comment]
sure.. but i do not expect that much conflicts

**Pull request merged**